CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 master
 ------
 
+- (`#303 <https://github.com/openscm/pymagicc/pull/303>`_) Refactor ``pymagicc.io`` into a package
```

Comment:
```suggestion
- (`#303 <https://github.com/openscm/pymagicc/pull/303>`_) Refactor ``pymagicc.io`` into multiple files
```

I'm not sure it's a package just cause it's in its own folder (not a big point but I think this use of language is more accurate)

Reply:
Python packages are any folder with an __init__.py, modules are files ending in `.py`

Reply:
oh right oops I confused module and sub-package